spec/sections/standard.adoc

```diff
@@ -176,14 +177,14 @@ input file size for scaling memory requirement of job.
 The Interface object is the primary member that describes the command arguments, environment variables,
 mounts, settings, inputs and outputs defined for a job.
 
-* The Interface object MAY have a member with the name `cmd`. The member's value MUST be a string specifying the
+* The Interface object MAY have a member with the name `command`. The member's value MUST be a string specifying the
```

Comment:
Should the command be renamed to indicate that it is the command arguments?

`The Seed command member can be treated as analogous to the Docker CMD statement.`
Is this true? I thought we were expecting Seed images to have a set entrypoint (which would be the real command) and we were just passing arguments.